hack/run-integration-tests.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+infomsg() {
+  echo "[INFO] ${1}"
+}
+
+TEST_SUITE="backend"
+
+# process command line args
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -ts|--test-suite)
+      TEST_SUITE="${2}"
+      if [ "${TEST_SUITE}" != "backend" -a "${TEST_SUITE}" != "frontend" -a "${TEST_SUITE}" != "frontend-multi-cluster" ]; then
+        echo "--test-suite option must be one of 'backend', 'frontend', or 'frontend-multi-cluster'"
+        exit 1
+      fi
+      shift;shift
+      ;;
+    -h|--help)
+      cat <<HELPMSG
+Valid command line arguments:
+  -ts|--test-suite <backend|frontend|backend-multi-cluster|frontend-multi-cluster|all>
```

Comment:
options `backend-multi-cluster` and `all` not yet implemented, but the script returns valid options so I see no problem here

Reply:
Whoops I'll clean that up in a follow up